Relativity.Pivot.SDK.md

```diff
@@ -0,0 +1,23 @@
+# Relativity.Pivot.SDK Version Compatibility
+
+[![nuget](https://img.shields.io/nuget/v/Relativity.Pivot.SDK.svg)](https://www.nuget.org/packages/Relativity.Pivot.SDK/)
+
+Relativity Pivot definitions for use with Kepler proxy generations.
+
+## v2.2.x
+
+### Release Notes
+
+* Initial public consumable SDK of Relativity Pivot APIs.
+
+### Supported Relativity Version Range
+
+Lowest Version | Highest Version
+--- | ---
+12.3.0.0 | Latest
+
+### Supported Pivot RAP Version Range
+
+Lowest Version | Highest Version
+--- | ---
+2.2.x | Latest
```

Comment:
2.2.x is not the standard for naming the nuget packages in the documentation

Reply:
thanks for changing to the exact version!